docs/guides/configurations.md

```diff
@@ -0,0 +1,402 @@
+
+
+## Notable values in Vald Helm Chart
+
+### Basics
+
+#### Specify image tag
+
+It is highly recommended to specify Vald version.
+You can specify image version by set `image.tag` field in each components (`[component].image.tag`) or `defaults` section.
+
+```yaml
+defaults:
+  image:
+    tag: v0.0.33
+```
+
+or you can use older image only for agent,
+
+```yaml
+agent:
+  image:
+    tag: v0.0.31
+```
+
+#### Specify appropriate logging level and format
+
+The default logging levels and formats are configured in `defaults.logging.level` and `defaults.logging.format`.
+You can also specify levels and formats in each component sections (`[component].logging`).
+
+```yaml
+defaults:
+  logging:
+    level: info
+    format: raw
+```
+
+you can specify log level `debug` and json format for gateway by the followings:
+
+```yaml
+gateway:
+  logging:
+    level: debug
+    format: json
+```
+
+#### Servers
+
+Each Vald components has several type of servers.
+They can be configured by specifying the values in `defaults.server_config` and can be overwritten by specifying `[component].server_config`.
+
+Examples:
+
+```yaml
+defaults:
+  server_config:
+    servers:
+      grpc:
+        enabled: true
+        host: 0.0.0.0
+        port: 8081
+        servicePort: 8081
+        server:
+          mode: GRPC
+          ...
+```
+
+```yaml
+gateway:
+  server_config:
+    servers:
+      rest:
+        enabled: true
+        host: 0.0.0.0
+        port: 8080
+        servicePort: 8080
+        server:
+          mode: REST
+          ...
+```
+
+##### gRPC server
+
+gRPC server should be enabled, because all Vald components are using gRPC to communicate with other Vald components.
+
+The API specs are placed in [apis/docs][vald-apis-docs].
+
+
+##### REST server
+
+REST server is optional.
+
+The swagger specs are placed in [apis/swagger][vald-swagger-specs].
+
+
+##### Health check servers
+
+There are two types of health check servers are built-in, liveness and readiness.
+They are used as servers for K8s liveness and readiness probe.
+
+By default, liveness servers are disabled for agent and compressor, because the liveness probes may cause to kill these components accidentally.
+
+```yaml
+agent:
+  server_config:
+    healths:
+      liveness:
+        enabled: false
+```
+
+##### Metrics servers
+
+Metrics servers are useful for debugging or monitoring Vald components.
+There are two types of metrics servers, pprof and prometheus.
+
+pprof server is a server that implemented using golang's net/http/pprof package.
+It exports profiling data that can be analyzed using [google's pprof][google-pprof] tool.
+
+prometheus server is just a [Prometheus][prometheus-io] exporter.
+To monitor Vald components by using prometheus server, it is needed to set `observability` section appropriately.
+
+#### Observability
+
+The observability features are useful for monitoring Vald components.
```

Comment:
```suggestion
The observability feature is useful for monitoring Vald components.
```

Reply:
Vald's observability features are not a single feature, they are composed of Prometheus exporter, Jaeger exporter, gRPC handler intercepter, tracer, etc... I meant them by writing "The observability features".
is it bad syntax? :thinking:

Reply:
I think you are correct. :)